modules/exploits/linux/http/f5_icontrol_rce.rb

```diff
@@ -0,0 +1,166 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = ExcellentRanking
+
+  include Msf::Exploit::Remote::HttpClient
+  prepend Msf::Exploit::Remote::AutoCheck
+
+  def initialize(info = {})
+    super(
+      update_info(
+        info,
+        'Name' => 'F5 BIG-IP iControl REST authentication bypass',
```

Comment:
It's an RCE, not only an auth bypass.

Reply:
I see what you're saying but the whole vulnerability is how to bypass the authentication to reach the `/mgmt/tm/util/bash` endpoint.

Reply:
Should rename this to something like "Authentication Bypass And RCE" in that case, would make more sense to end users.

Reply:
Perhaps "RCE via Authentication Bypass" would reflect this better?

Reply:
The filename still needs to change, too. It's too generic at the moment.

Reply:
```
wvu@hiigara:~/metasploit-framework:HEAD$ search f5_icontrol modules
modules/exploits/linux/http/f5_icontrol_rest_ssrf_rce.rb
modules/exploits/linux/http/f5_icontrol_exec.rb
modules/exploits/linux/http/f5_icontrol_rce.rb
wvu@hiigara:~/metasploit-framework:HEAD$
```